docs/section23/Rule23-3.md

```diff
@@ -22,33 +22,46 @@
+        - create a variable to hold the total minimum volume flow rate of all terminals in the proposed zone.  This will be an array of values, one for each hour of the year.  We will sum the minimum volume flowrate for each terminal by the availability of the terminal to take into account cases where the proposed zone has multiple terminals that do not operate at the same time: `minimum_volume_list_p = []`
+        - look at each terminal in the proposed zone: `for terminal_p in zone_p.terminals:`
+          - get the HVAC system associated with the terminal: `hvac_p = terminal_p.served_by_heating_ventilating_air_conditioning_system`
+          - get the fan system: `fan_system_p = hvac_p.fan_system`
+          - get the operation schedule hourly values: `operation_schedule_hourly_values_p = fan_system_p.operating_schedule.hourly_values`
+          - get the terminal minimum volume flowrate: `min_volume_p = terminal_p.minimum_airflow`
+          - multiply each value in the hourly schedule by the minimum airflow rate, and append this to the list of minimum volume flowrates: `minimum_volume_list_p.append([i * min_volume_p for i in operation_schedule_hourly_values_p]`
+        - sum all of the values for each hour, so that the resulting list is a list of the sum of minimum flowrates across all terminals each hour of the year: `min_volume_flow_tot_p = [sum(values) for values in zip(*minimum_volume_list_p)]`
+        - find the maximum value in the list: `min_volume_p = max(min_volume_flow_tot_p)`
+
+        **Rule Assertion:**    
+
+        - Case 1: For each terminal that is served by HVAC system that is Type-5, 7, 7a, 5b, 7b, 7c, 6, 8, 8a, 6b, 8b, if minimum volume setpoint is equal to the maximum of: 30% of peak primary design airflow, the rate required for minimum outside air, or the proposed minimum volume airlfow, PASS: `if terminal_b.minimum_airflow == MAX(terminal_b.primary_airflow * 0.3, terminal_b.minimum_outdoor_airflow, minimum_volume_p): PASS`
+       
+        - Case 2: Else, if the minimum volume setpoint is equal to the maximum of 30% of peak primary design airflow, the rate required for minimum outside air, then UNDETERMINED with a note.  Due to the last check, this case indicates that the minimum volume flow is less than the proposed minimum airflow for the same zone.  Indicate to the AHJ to check if minimum airlfow in the baseline complies with local standards: `elif terminal_b.minimum_airflow == MAX(terminal_b.primary_airflow * 0.3, terminal_b.minimum_outdoor_airflow): UNDETERMINED; note = "The minimum volume flowrate is equal to the maximum of 30% of the terminal primary airflow rate and the terminal minimum outdoor airflow, but it is less than the minimum airflow in the proposed design.  Check that the baseline minimum airlfow is sufficient to comply with accredidation standards."`
```

Comment:
I think this hsould be if terminal_b.minimum_airflow == terminal_b.primary_airflow * 0.3: UNDETERMINED

Reply:
Case 1 has three conditions - the ones related to the baseline system, and the one related to the proposed.  In case 2, we're looking to see if `terminal_b.minimum_airflow` is equal to one of the cases related to the baseline system, but greater than the `minimum_volume_p`.  We don't need to do the `if terminal_b.minimum_airflow > minimum_volume_p` check here because it's already covered by the case 1 check, which evaluted to false.